src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -90,6 +90,12 @@ public bool ErrorEndLocation
             get { return (string?)_store[nameof(GeneratedFilesOutputPath)]; }
         }
 
+        public string? GeneratedArtifactsOutputPath
```

Comment:
Is this hooked up in the task file anywhere?

Reply:
what is "the task file"?  thanks!

Reply:
See https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L71

Reply:
done.